packages/graphiql/src/components/__tests__/helpers/codeMirror.js

```diff
@@ -0,0 +1,81 @@
+export function MockCodeMirror(node, { value, ...options }) {
```

Comment:
also the jest convention is to use __mocks__ directory I think:
https://jestjs.io/docs/en/manual-mocks#mocking-user-modules

however the `codeMirrorModules` const export maybe belongs in helpers?